src/redux/stock/helper.js

```diff
@@ -0,0 +1,44 @@
+const processMonthlyData = (data, range) => {
+  // we need to group and calculate the avg by months
+  const key2Data = {};
```

Comment:
The name 'key2Data' and 'key2Avg' is ambiguous, please give them more clear names.